src/CMakeLists.txt

```diff
@@ -14,12 +14,26 @@ set_target_properties(rade PROPERTIES
 )
 
 add_executable(radae_tx radae_tx.c)
-target_link_libraries(radae_tx rade Python3::Python)
+target_link_libraries(radae_tx rade radecore opus m Python3::Python)
 target_include_directories(radae_tx PRIVATE
     "$<TARGET_PROPERTY:Python3::NumPy,INTERFACE_INCLUDE_DIRECTORIES>")
 
 add_executable(radae_rx radae_rx.c)
-target_link_libraries(radae_rx rade Python3::Python)
+target_link_libraries(radae_rx rade radecore opus m Python3::Python)
 target_include_directories(radae_rx PRIVATE
     "$<TARGET_PROPERTY:Python3::NumPy,INTERFACE_INCLUDE_DIRECTORIES>")
 
+add_library(radecore rade_enc.c rade_dec.c rade_enc_data.c rade_dec_data.c)
```

Comment:
Does it make sense to have a separate `radecore` library just for the C port? There'd have to be modifications on the freedv-gui side to also make sure `libradecore.dll` got properly packaged along with `librade.dll`.

Reply:
Sure now the port is done I can reconcile into one library.

Reply:
https://github.com/drowe67/radae/pull/33/commits/8aec5a798eb278551faff8bee9c37cabccddeb81